src/relayserver/RegistrationManager.ts

```diff
@@ -261,6 +257,7 @@ export class RegistrationManager {
     const addRelayWorkerMethod = await this.contractInteractor.getAddRelayWorkersMethod([this.workerAddress])
     const details: SendTransactionDetails = {
       signer: this.managerAddress,
+      serverAction: ServerAction.ADD_WORKER,
```

Comment:
noticed you added action type, but I didn't see anywhere it is used.

Reply:
Fixed.